mod.config

```diff
@@ -4,7 +4,8 @@
 
 MOD_ID="ra2"
 
-ENGINE_VERSION="049ed08"
+# HACK HACK HACK: Use a custom engine which reverts https://github.com/OpenRA/OpenRA/pull/14043
+ENGINE_VERSION="147557e"
```

Comment:
This is making the mod depend on an incompatible fork. The regression in https://github.com/OpenRA/OpenRA/pull/14043 should be investigated instead.

Reply:
Alternatively copy in the old WavLoader into this mod's codebase.

Reply:
What do you mean by incompatible fork? The branch can be rebased the same like you would rebase a PR. I would even go that far to call getting conflicts during that a feature, making you aware of changes that would otherwise go under the radar (like the list/cue PR). (I don't like the idea of introducing our own WavLoader, although I won't really argue against it if you insist.)
In any case: I don't think there is a way around using a custom fork for now. Even if the bug is fixed upstream, we would still need to cherry-pick the fix onto the release tag as we can't use commits between releases anymore with the new updater system.